src/ResourceManagement/MachineLearning/Microsoft.Azure.Management.MachineLearning/generate.cmd

```diff
@@ -8,7 +8,7 @@ setlocal
 
 set autoRestVersion=0.17.0-Nightly20160824
```

Comment:
We would still prefer if you updated this file with the version of AutoRest used (even if it's from the new instructions); it let's us know that you aren't using an outdated version of AutoRest if we were to just look at this file